java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -2264,7 +2264,7 @@ public void testZstd() throws Exception {
         OrcFile.writerOptions(conf)
             .setSchema(schema)
             .compress(CompressionKind.ZSTD)
-            .bufferSize(1000)
+            .bufferSize(2000)
```

Comment:
Hi, @williamhyun . I'll revive this test case. It turns out that this is a valid test coverage.